dicomanonymizer/simpledicomanonymizer.py

```diff
@@ -104,10 +104,6 @@ def empty(dataset, tag):
 def deleteElement(dataset, element):
     if element.VR == 'DA':
         replaceElementDate(element)
-    elif element.VR == 'SQ':
-        for subDataset in element.value:
-            for subElement in subDataset.elements():
-                deleteElement(subDataset, subElement)
     else:
         del dataset[element.tag]
```

Comment:
I think the expected behavior of `deleteElement` is now not clear.
It does not delete 'DA' elements, but it does delete them if they are nested inside 'SQ' elements (because 'SQ' elements are deleted).

Why don't you also delete 'DA' elements ?

Reply:
That was the behavior before I took the project. I didn't check all actions methods.
I'll check the documentation about this.

Reply:
So, that's a good point.
Some tags like (0040,0244) have different actions. The basic approach is to delete it, BUT, it has to be keep if it's a non-sequence attributes. If it's a sequence attribute, then it has to be cleaned. But cleaning a DA is to replace the value with a similar meaning value...
There are a lot of subtilities like this in the official documentation :
http://dicom.nema.org/dicom/2013/output/chtml/part15/chapter_E.html#table_E.1-1

So please don't merge. I'll have to still work on it.